### ISSUE TITLE
Name clash with SOLID_FRACTION variable

### ISSUE THREAD

**Issue by jcotela** [KratosMultiphysics/Kratos#428]
I have encountered a problem with the following variables
- SOLID_FRACTION added in dem_variables.h
- SOLIDFRACTION added in c2c_variables.h
The problem is that the SOLIDFRACTION variable does not use the standard register macro, and is registered with the custom string "SOLID_FRACTION". As a result, python gets confused. This can be seen by doing
```
from KratosMultiphysics import *
print(SOLID_FRACTION)
print(SOLIDFRACTION)
```
which produces:
```
 |  /           |             
 ' /   __| _` | __|  _ \   __|
 . \  |   (   | |   (   |\__ \ 
_|\_\_|  \__,_|\__|\___/ ____/
           Multi-Physics 5.0.0-19443f6d2
SOLID_FRACTION variable #0
SOLID_FRACTION variable #723
```
If I try to assing the SOLID_FRACTION variable later in the python script, it complains because the variable is unregistered/has key 0.

I would suggest using the string "SOLID FRACTION" (no underscores) to register the C2C variable, avoiding the name clash. Would that be OK, @jrubiogonzalez?

By the way, the C2C variables SHRINKAGE_POROSITY and SHRINKAGE_POROSITY_US have the same problem (they are both registered to "SHRINKAGE_POROSITY").

I'm tagging @maceligueta because I think this might be a problem for the DEM application too.

**Comment by maceligueta** [comment]
Thank for informing, @jcotela !
Why not use the standard form? KRATOS_CREATE_VARIABLE(SOLIDFRACTION)

**Comment by jcotela** [comment]
I'm not sure about this case, but these non-standard variable definitions are normally done to have "nice" names in the postprocess file (like "velocity (m/s)" instead of "VELOCITY"). It is not a recommended practice, however.

**Comment by maceligueta** [comment]
good to know!

**Comment by pooyan-dadvand** [comment]
Indeed it is an error, however having both of them in core avoids key confilcts. We should have a unit field for variable to avoid such a definition. @jrubiogonzalez was also working on this.

**Comment by jcotela** [comment]
Note that the SOLID_FRACTION variable was removed from the dem variable list in PR #448. I have added it back (as a deprecated variable) to ensure compilation of the trilinos application in PR #452, but I'm willing to remove it for good once we reach a decision on this.

**Comment by RiccardoRossi** [comment]
@pablobecker Cold you take  a look?

**Comment by jcotela** [comment]
@RiccardoRossi The main issue for us is that `SOLID_FRACTION` is still used in the ThermoMechanicalApplication. If @jrubiogonzalez or @pablobecker give me the OK I can just remove it for good.

**Comment by RiccardoRossi** [comment]
@jcotela, that s why i pimged Pavlo 😋

**Comment by pablobecker** [comment]
Ok, I'll ask Jordi tomorrow.

**Comment by jrubiogonzalez** [comment]
I changed to SOLIDFRACTION in the register also, and all the registers are standard now, so no clash should be happening. Could you modify it? I have it a little bit harder to upload to Git.

**Comment by pooyan-dadvand** [comment]
@pablobecker would you please address this issue before the release branch? I have just add it to the release milestone.

**Comment by pooyan-dadvand** [comment]
@pablobecker, @jrubiogonzalez , @ddiezrod would you please address this issue?

**Comment by ddiezrod** [comment]
I will do it now

**Comment by pooyan-dadvand** [comment]
Thanks! :+1:

**Comment by ddiezrod** [comment]
@jcotela  @pooyan-dadvand  I solved this in #900 ,  please someone review it

**Comment by ddiezrod** [comment]
I'll close this issue
